frontend/src/components/editor/Editor.tsx

```diff
@@ -113,6 +286,14 @@ function Editor() {
 						minHeight: "100%",
 					}}
 				/>
+				{formatBarState.show && editorStore.cmView && (
```

Comment:
Because the type of `editorStore.cmView` is `EditorView | null`, the result of `formatBarState.show && editorStore.cmView` can be `null`. (When `formatBarState.show == true`, `editorStore.cmView == null`)

So it should be wrapped with `Boolean`.

```typecript
Boolean(formatBarState.show && editorStore.cmView)
```